DESCRIPTION

```diff
@@ -1,19 +1,23 @@
 Package: Siane
-Title: Siane
+Title: Explore the Spanish INE Data using this Geographic Tool
 Version: 0.1
-Description: Statistic polygon plots of INE data 
-Depends: R (>= 3.3.1)
-License: MIT
+Description: A tool that assists the user in two tasks: the map reading process and the
+  data representation process. The reading process is leveraged with functions that allow
+  the user to find maps in the Siane repository easily, i.e. searching by map year or
+  administrative level. The other main functionality is to bind numerical data to polygons.
+  This package is only compatible with the spanish IGN(National Institute of Geography) maps
+  and the INE(National Institute of Statistics) data.
+Depends: plyr, raster, rgdal, sp, methods, utils
+License: MIT + file LICENSE
 Encoding: UTF-8
 LazyData: true
 Authors@R: c(person("Carlos J. Gil Bellosta",
-  email = "cgb@datanalytics.com",role = c("aut","cre")),
-  person("Nuno", "Carvalho", role = c("aut"),
+  email = "cgb@datanalytics.com", role = c("aut")),
+  person("Nuno", "Carvalho", role = c("cre", "aut"),
   email = "nunocarvalhodossantos@gmail.com"))
-Author: Carlos J. Gil Bellosta[aut, cre]
-  Nuno Carvalho[aut]
+Author: Carlos J. Gil Bellosta [aut],
```

Comment:
This field, as well as the `Maintainer` field, can be dropped if `Authors@R` is present, as it is the case. Advantages: remove duplications, easier to maintain.